qiita_pet/handlers/download.py

```diff
@@ -374,6 +378,128 @@ def get(self, path):
         self.finish()
 
 
+class DownloadDataReleaseFromPrep(BaseHandlerDownload):
+    @authenticated
+    @coroutine
+    @execute_as_transaction
+    def get(self, prep_template_id):
```

Comment:
it might be nice to annotate such a large method with some comments, although I think much of it is fairly self-explanatory (to me at least.)

Reply:
To me too!